project/pom.xml

```diff
@@ -572,23 +572,6 @@
                 <package>com.google</package>
               </packages>
             </exception>
-            <exception>
```

Comment:
Doesn't seem to be necessary any more? Ran fine locally, at least - let's see what the PR builder has to say...